sdk/nodejs/runtime/invoke.ts

```diff
@@ -254,6 +254,7 @@ async function createInvokeRequest(
     req.setArgs(obj);
     req.setProvider(provider || "");
     req.setVersion(opts.version || "");
+    req.setPlugindownloadurl(opts.pluginDownloadURL || "")
```

Comment:
What would be the best way to test this?

Reply:
Conformance testing probably, we should give a think to what that looks like cause it'll need a bit of work. Provider loading is pretty hard coded into the engine at the moment.